app/views/ImportGoodsView.scala.html

```diff
@@ -28,6 +28,9 @@
 
     @formHelper(action = routes.ImportGoodsController.onSubmit(mode), 'autoComplete -> "off") {
 
+         <p class="govuk-caption-xl hmrc-caption-x"><span class="govuk-visually-hidden">
+            This section is </span>@messages("importGoods.caption")</p>
```

Comment:
needs to go to messages file // welsh translation

Reply:
https://github.com/hmrc/advance-valuation-rulings-frontend/pull/7/commits/580660c4f141b67f2b09e5aa5e8a75198b8d916d